repos/fdbt-site/src/pages/api/manageOperatorDetails.ts

```diff
@@ -5,7 +5,7 @@ import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
 import { GS_OPERATOR_DETAILS_ATTRIBUTE } from '../../constants/attributes';
 import { removeExcessWhiteSpace } from '../../utils/apiUtils/validator';
 import { upsertOperatorDetails } from '../../data/auroradb';
-import { OperatorDetails } from 'shared/matchingJsonTypes';
+import { OperatorDetails } from '../../../shared/matchingJsonTypes';
```

Comment:
This is wrong now

Reply:
Trying to figure out why - my IDE says its wrong unless it has the full path